test/functional/sqlsrv/MsCommon.inc

```diff
@@ -84,18 +84,6 @@ function isDaasMode()
     return ($daasMode ? true : false);
 }
 
-// function isAEQualified($conn)
```

Comment:
don't we have tests which need this??

Reply:
the method has been moved to MsHelper in the AE namespace